### ISSUE TITLE
accury problem ade20k

### ISSUE THREAD

**Issue by zxmyyds** [CoinCheung/BiSeNet#296]
![1679707196647](https://user-images.githubusercontent.com/103728486/227675438-57c44421-2041-4029-9e3c-96d9e15aadd3.png)
I trained ADE20k, but why the accuracy makes such a difference

**Comment by CoinCheung** [comment]
What modifications did you made? And which dataset did you use? Would you please tell me how did you reach this result?

**Comment by zxmyyds** [comment]
@CoinCheung I did not make any modification and trained ade20k 2016 according to the instructions. I also wonder why some training shows 0. Have you encountered such a situation

**Comment by CoinCheung** [comment]
Then tell me what is your environment is like, and the steps following which I can achieve your result?

**Comment by zxmyyds** [comment]
![1679709829827](https://user-images.githubusercontent.com/103728486/227680354-f869143e-07b9-4e94-803a-f07684fa71b8.png)
![1679709897834](https://user-images.githubusercontent.com/103728486/227680355-7a7351d3-4f29-4efd-aed1-87e6c1c5fb4f.png)
I just did these steps

**Comment by CoinCheung** [comment]
Maybe you should adjust your batch size to match the configuration of what I used. I used 8 gpus, but you only used one. 

If your gpu memory cannot afford such batch size, you should try to reduce learning rate.

The training parameters I provided is not carefully tuned(because authors did not have such experiments in the paper), maybe you can have a round of grid search to find the best configuration.

**Comment by zxmyyds** [comment]
Thanks I used the city training set and its accuracy is normal and I need to try it again

**Comment by CoinCheung** [comment]
Maybe you have achieved the results that you see correct, so I am closing this.

**Comment by XU-GITHUB-curry** [comment]
> Thanks I used the city training set and its accuracy is normal and I need to try it again

I met the same problem on one single GPU without any modification, did you solve the problem? thank you
